### PR TITLE
feat(CenterPoint): Release CenterPoint v1.7 and CenterPoint v1.7.1

### PR DESCRIPTION
**Pull request by KSeangTan**
## Summary
For `CenterPoint v1.7`, we introduce the following changes:
    - Add label for "construction_vehicle": "truck"
    - Introduce `db_j6gen2_v1_1`
    - Add the feature of distance between of each pointcloud to center of pillar in Z

For `CenterPoint v1.7.1`, we introduce the following changes:
    - Add a dataset of `j6gen2_base`
    - Finetune `j6gen2` model base on `CenterPoint v1.7` with intensity 
  

#### base
| eval range: 120m         | mAP  | car <br> (90,242) | truck <br> (14,910) | bus <br> (4,992) | bicycle <br> (4,666) | pedestrian <br> (36,690) |
| -------------------------| ---- | ----------------- | ------------------- | ---------------- | -------------------- | ------------------------ |
| CenterPoint base/1.6      | 67.96 | 81.09              | 54.12                | 80.55             | 58.48                 | 65.57                     |
| CenterPoint base/1.7 (A)    | 66.92 | 81.06              | **54.49**                | 77.46             | 56.11                 | 65.48                     |
| CenterPoint base/1.7 (B)     | **68.21** | **81.13**              | 53.95                | **80.97**             | **59.19**                 | **65.79**                     |

#### J6gen2
| eval range: 120m         | mAP  | car <br> (28,002) | truck <br> (1,123) | bus <br> (1,203) | bicycle <br> (223) | pedestrian <br> (4,007) |
|------------------------|-------|--------|--------|--------|---------|-------------|
| CenterPoint base/1.6        | 72.20 | 86.17 | 52.62 | 85.53 | 74.99   | 61.69       |
| CenterPoint base/1.7 (A)    | 71.04 | 85.94 | 53.53 | 84.91 | 69.01   | 61.81       |
| CenterPoint base/1.7 (B)    | 72.29 | 85.96 | 53.52 | 84.21 | 74.96   | 62.79       |
| CenterPoint base/1.7.1 (C)  | 72.94 | 86.96 | 57.88 | **85.09** | 71.62   | 63.17       |
| CenterPoint base/1.7.1 (D)  | **74.64** | **87.87** | **59.87** | 84.50 | **77.35**   | **63.62**       |

| eval range: 120m         | mAP  | car <br> (82,227) | truck <br> (10,662) | bus <br> (4,648) | bicycle <br> (4,246) | pedestrian <br> (333,03) |
| -------------------------| ---- | ----------------- | ------------------- | ---------------- | -------------------- | ------------------------ |
| CenterPoint base/1.7.1 (D)     | 68.21 | 81.13              | 53.95                | 80.97             | 59.19                 | 65.79                     |
| CenterPoint base/1.7.1 (C)    | 66.92 | 81.06              | 54.49                | 77.46             | 56.11                 | 65.48                     |
| CenterPoint base/1.7 (B)     | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.7 (A)    | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.6      | 66.6 | 80.3              | 54.5                | 79.5             | 55.1                 | 63.5                     |

#### largebus
| eval range: 120m         | mAP  | car <br> (82,227) | truck <br> (10,662) | bus <br> (4,648) | bicycle <br> (4,246) | pedestrian <br> (333,03) |
| -------------------------| ---- | ----------------- | ------------------- | ---------------- | -------------------- | ------------------------ |
| CenterPoint base/1.7.1 (D)     | 68.21 | 81.13              | 53.95                | 80.97             | 59.19                 | 65.79                     |
| CenterPoint base/1.7.1 (C)    | 66.92 | 81.06              | 54.49                | 77.46             | 56.11                 | 65.48                     |
| CenterPoint base/1.7 (B)     | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.7 (A)    | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.6      | 66.6 | 80.3              | 54.5                | 79.5             | 55.1                 | 63.5                     |

#### JPNTaxi
| eval range: 120m         | mAP  | car <br> (82,227) | truck <br> (10,662) | bus <br> (4,648) | bicycle <br> (4,246) | pedestrian <br> (333,03) |
| -------------------------| ---- | ----------------- | ------------------- | ---------------- | -------------------- | ------------------------ |
| CenterPoint base/1.7 (B)     | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.7 (A)    | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.6      | 66.6 | 80.3              | 54.5                | 79.5             | 55.1                 | 63.5                     |

#### J6
| eval range: 120m         | mAP  | car <br> (82,227) | truck <br> (10,662) | bus <br> (4,648) | bicycle <br> (4,246) | pedestrian <br> (333,03) |
| -------------------------| ---- | ----------------- | ------------------- | ---------------- | -------------------- | ------------------------ |
| CenterPoint base/1.7 (B)     | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.7 (A)    | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.6      | 66.6 | 80.3              | 54.5                | 79.5             | 55.1                 | 63.5                     |

Where (A) and (B) represent model without and with the feature of distance between of each pointcloud to center of pillar in Z, respectively.

#### largebus
| eval range: 120m         | mAP  | car <br> (82,227) | truck <br> (10,662) | bus <br> (4,648) | bicycle <br> (4,246) | pedestrian <br> (333,03) |
| -------------------------| ---- | ----------------- | ------------------- | ---------------- | -------------------- | ------------------------ |
| CenterPoint base/1.7.1 (B)     | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.7.1 (A)    | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.6      | 66.6 | 80.3              | 54.5                | 79.5             | 55.1                 | 63.5                     |

Where (A) and (B) represent model finetuned from CenterPoint base/1.7 (B) without and with intensity, respectively.

#### J6gen2
| eval range: 120m         | mAP  | car <br> (82,227) | truck <br> (10,662) | bus <br> (4,648) | bicycle <br> (4,246) | pedestrian <br> (333,03) |
| -------------------------| ---- | ----------------- | ------------------- | ---------------- | -------------------- | ------------------------ |
| CenterPoint base/1.7.1 (B)     | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.7.1 (A)    | 67.9 | 81.1              | 55.1                | 80.4             | 57.4                 | 65.6                     |
| CenterPoint base/1.6      | 66.6 | 80.3              | 54.5                | 79.5             | 55.1                 | 63.5                     |

Write what you change and why you change

## Change point

Explain for detail

If you add new feature or fix for core library

- feat(autoware-ml): add {feature name}
- fix(autoware-ml): fix {fixed point}

If you add new feature or fix for projects

- feat(BEVFusion): add {feature name}
- feat(BEVFusion): add config file
- fix(BEVFusion): fix {fixed point}

If you change the thing without program

- chore(BEVFusion): add README.md

## Note

Write note like influences for other projects.

## Test performed

- Log

```
Log output
```

### CODE REVIEW

**Review by scepter914 — COMMENTED** [review]
Thank you for great contribution.
The result looks so great and the contribution of (data + adding layer) and the fine-tuning techniques is well organized.
The report is very clear and easy to understand.

I leave comments on config of T4dataset.
Would you confirm it?

File: autoware_ml/configs/t4dataset/db_j6gen2_v1_1.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+version: 1
```
[the remark this answers is not in the document]
Comment (KSeangTan) [reply autoware_ml/configs/t4dataset/db_j6gen2_v1_1.yaml]: Make sense, I have updated `db_j6gen2_v1` instead, would you mind checking again? Thanks

**Review by scepter914 — APPROVED** [review]
I update the version of T4dataset at [c4a1c06](https://github.com/tier4/AWML/pull/56/commits/c4a1c06514cf2e3b8468271d4a67ce2cd8ba5bff).
Anyway, the PR looks great to me and this PR can be merged :+1:.
